[android] parse-jacoco-report.py is failing the build

Since #15434, the CI build on `master` is failing both `android-debug-arm-v8` and `android-gnustl-arm-v7` jobs with:

```
> Task :MapboxGLAndroidSDK:jacocoTestReport
Creating Jacoco Report for 1 coverage files
/src/platform/android/MapboxGLAndroidSDK/build/jacoco/testDebugUnitTest.exec

BUILD SUCCESSFUL in 8s
28 actionable tasks: 1 executed, 27 up-to-date
python platform/android/scripts/parse-jacoco-report.py
Traceback (most recent call last):
  File "platform/android/scripts/parse-jacoco-report.py", line 18, in <module>
    missed = int(numbers.group(1))
AttributeError: 'NoneType' object has no attribute 'group'
Makefile:763: recipe for target 'android-parse-and-send-jacoco-report' failed
make: *** [android-parse-and-send-jacoco-report] Error 1
Exited with code 2
```

/cc @mapbox/maps-android

--- a/platform/android/gradle/jacoco-report.gradle
+++ b/platform/android/gradle/jacoco-report.gradle
@@ -15,7 +15,7 @@ task jacocoTestReport(type: JacocoReport, dependsOn: ['testDebugUnitTest']) {
     }
 
     def fileExcludes = ['**/R.class', '**/R$*.class', '**/BuildConfig.*', '**/Manifest*.*', '**/*Test*.*', 'android/**/*.*']
-    def debugTree = fileTree(dir: "${project.buildDir}/intermediates/javac/debug/compileDebugJavaWithJavac/classes", excludes: fileExcludes)
+    def debugTree = fileTree(dir: "${project.buildDir}/intermediates/javac/debug/classes", excludes: fileExcludes)
     def mainSrc = "${project.projectDir}/src/main/java"
     println(mainSrc)
     def ecSrc = fileTree(dir: "$project.buildDir", include: "**/*.ec")

